### DISCUSSION TITLE
Versione 1.6.0

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#792] [Richieste di aiuto] [answered]
Salve potrei sapere quali saranno le novità della versione 1.6.0?

Grazie

**Comment by trinko** [comment, marked as answer]
Essenzialmente è un aggiornamento importante (e complicato) su tutto il codice per renderlo compatibile con il PHP 8.2 e la versione 6.4 del framework Symfony. 

Poi vedo se riesco ad inserire altro.
